language-support/ts/daml-ledger/index.test.ts

```diff
@@ -5,19 +5,20 @@ import { Template, Choice, ContractId } from "@daml/types";
 import Ledger, {CreateEvent} from "./index";
 import { Event } from "./index";
 import * as jtv from "@mojotech/json-type-validation";
-import type { EventEmitter } from 'events';
+import { EventEmitter } from 'events';
```

Comment:
Why? The changes below don't need this.

Reply:
that was just for me when debugging. somehow my IDE doesn't like this. I'll change it back when ready.